File: net/pdfjet/A3.cs
/*
 * A3.cs
 *
 * Copyright (c) 2026 PDFjet Software
 * Licensed under the MIT License. See LICENSE file in the project root.
 */
using System;

namespace PDFjet.NET {
/// <summary>
/// Used to specify PDF page with size <strong>A3</strong>.
/// For more information about the page size classes - A3, A4, A5, B5, JISB5, Executive, Letter, Legal and Tabloid - see the Page class.
/// </summary>
public class A3 {
    /// <summary>The A3 page size in portrait orientation.</summary>
    public static readonly PageSize PORTRAIT = new PageSize(842.0f, 1191.0f);
    /// <summary>The A3 page size in landscape orientation.</summary>
    public static readonly PageSize LANDSCAPE = new PageSize(1191.0f, 842.0f);
}
}   // End of namespace PDFjet.NET

File: src/a3/a3.go
// a3.go
//
// Copyright (c) 2026 PDFjet Software
// Licensed under the MIT License. See LICENSE file in the project root.

// Package a3 defines the A3 page size in points.
package a3

import "github.com/edragoev1/pdfjet/v9/src/pagesize"

// Portrait returns the A3 page size in portrait orientation.
func Portrait() pagesize.PageSize {
	return pagesize.NewPageSize(842.0, 1191.0)
}

// Landscape returns the A3 page size in landscape orientation.
func Landscape() pagesize.PageSize {
	return pagesize.NewPageSize(1191.0, 842.0)
}
